x-pack/plugins/uptime/e2e/page_objects/login.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { Page } from '@elastic/synthetics';
+
+export function loginPageProvider({ page }: { page: Page; kibanaUrl: string }) {
+  return {
+    async waitForLoadingToFinish() {
+      while (true) {
+        if ((await page.$('[data-test-subj=kbnLoadingMessage]')) === null) break;
+        await page.waitForTimeout(5 * 1000);
```

Comment:
i think we should reduce this timeout , WDYT? 5 seconds seems too slow for some requests.

Reply:
I think higher timeouts will reduce flakiness, and this isn't outrageously high that I feel it'll add an outrageous amount of time to run the tests.